trino-parser/src/main/java/io/trino/sql/ExpressionFormatter.java

```diff
@@ -153,7 +153,7 @@ protected String visitNode(Node node, Void context)
         @Override
         protected String visitRow(Row node, Void context)
         {
-            String rowPrefix = (dialect == DEFAULT || dialect == BIGQUERY) ? "ROW" : "";
+            String rowPrefix = "";
```

Comment:
There are some issues with formatting Trino SQL. Consider the followed case:
```sql
select sum(c1) from (values (1,2), (2,3)) t(c1, c2)
```
This SQL will be formatted to 
```sql
select sum(c1) from (values ROW(1,2), ROW(2,3)) t(c1, c2)
```
It's valid for Trino but can't be handled correctly when transpiring to the BigQuery sql by sqlglot.
The transpiring result would like
```sql
SELECT SUM(`c1`) FROM UNNEST([STRUCT(STRUCT(1, 2) AS `c1`), STRUCT(STRUCT(2, 3) AS `c1`)])
```
It's wrong. It should be
```sql
SELECT SUM(`c1`) FROM UNNEST([STRUCT(1 AS `c1`, 2 AS `c2`), STRUCT(2 AS `c1`, 3 AS `c2`)])
```